app/controllers/api/v1/authentication_controller.rb

```diff
@@ -62,13 +62,14 @@ def send_pin
         end
 
         @user = User.find_by_mobile(@mobile.mobile)
+        @otp_auth_key = otp_auth_key_for(@user, refresh: true)
 
         if @user && @user.allowed_login?(app_name)
           @user.send_verification_pin(app_name, params[:mobile])
         elsif @user
           return render json: {error: "You are not authorized."}, status: 401
         end
-        render json: {otp_auth_key: otp_auth_key_for(@user)}
+        render json: {otp_auth_key: @otp_auth_key }
```

Comment:
Just a small change, space missing in braces
render json: { otp_auth_key: @otp_auth_key }